torch/csrc/jit/interpreter.cpp

```diff
@@ -213,46 +485,83 @@ Operation getOperation(jit::Node *node) {
     }
   IR_ELSEIF(FusionGroup)
     auto fusion_fn = sharedFusionCompiler().getOrCompile(value);
-    return [fusion_fn](const list_of_retainable & inputs, list_of_retainable & outputs) {
+    auto num_inputs = value->inputs().size();
+    return [fusion_fn, num_inputs](Stack & stack) {
       autograd::profiler::RecordFunction record("FusionGroup");
-      tensor_list tinputs = unsafeToTensorListShare(inputs);
-      tensor_list toutputs;
-      fusion_fn->launch(tinputs, toutputs);
-      moveToListOfRetainables(std::move(toutputs), outputs);
+      Stack toutputs;
```

Comment:
use `std::vector<...>` directly? It's not like we're using any of the `Stack` functions, so it's nicer to keep the type alias meaningful